app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def bootstrap_class_for(flash_type)
+    {
+      success: "alert-success",
+      error: "alert-danger",
+      alert: "alert-warning",
```

Comment:
I think that Devise uses the "alert" type for errors. When I try to log in without entering a password, the message is styled yellow but when I try to change my password with a missing password, the message is styled red.

Reply:
I'll submit an issue for this one.

Reply:
✔️

Reply:
✔️